### ISSUE TITLE
Silently fails to load .env with multiline variables

### ISSUE THREAD

**Issue by shanesmith** [DarthSim/overmind#172]
If a `.env` contains a multiline variable overmind will silently fail to load the whole file.

```sh
$ cat .env
FOO='foo'
BAR='
multi
line
'

$ overmind run bash -c 'echo "Values: $FOO $BAR"'
Values:
```

**Comment by shanesmith** [comment]
Thanks for the very prompt attention to this!

We currently install from the binaries, any chance this would trigger a release?

**Comment by Envek** [comment]
Probably we will make a release later this month.

**Comment by Envek** [comment]
Released in [2.5.0](https://github.com/DarthSim/overmind/releases/tag/v2.5.0), please enjoy.

**Comment by shanesmith** [comment]
Much appreciated!
